guides/adding_examples_to_your_app.md

```diff
@@ -0,0 +1,77 @@
+# Adding Examples To Your App
+
+Docs: examples
+
+## Introduction
+
+As we saw in the [Quickstart](/getting_started#example-inputs), providing example inputs to your app are a helpful way
+to let your audience explore the functionality of your app without having them specify all of your app's expected inputs.
+
+In this guide we'll go into greater depth on how you can provide example inputs to your apps.
+
+## Adding examples to an Interface
+
+As covered in the Quickstart, adding examples to an Interface is as easy as providing a list of lists to the `examples`
+keyword argument. 
+Each sublist is a data sample, where each element corresponds to an input of the prediction function.
+The inputs must be ordered in the same order as the prediction function expects them.
```

Comment:
This is great @freddyaboulton! I would add a couple of things here:
* if the `Interface` has only 1 input component, then you can provide the examples as a regular list instead of a nested sublist
* you can also provide a path to a directory of examples

(Both of these are less common use cases, but I think we should include them in a Guide on examples for completeness).